### ISSUE TITLE
Buying from shop when worldserver is down

### ISSUE THREAD

**Issue by lingxDEV** [azerothcore/acore-cms#92]
Currently it's possible to buy from the shop while the worldserver is down.

Using mycred -- this means that the user is charged, but never recieves the item.

Would it be possible to only allow purchases when the worldserver is up?

**Comment by Helias** [comment]
similar issue to https://github.com/azerothcore/acore-cms/issues/76

**Comment by Goatform** [comment]
The best way for this would be to have an additional table and service for this that periodically checks for the data (items) in the table and process them to worldserver, this way in case worldserver is offline for some reason, it won't process it until it gets a response from the world.

**Comment by lingxDEV** [comment]
I agree that would be optimal. However, I think an error should be thrown on the frontend for now, as it's not as complex to achieve.
